oximeter/db/src/client.rs

```diff
@@ -1339,6 +1429,126 @@ mod tests {
         db.cleanup().await.expect("Failed to cleanup ClickHouse server");
     }
 
+    // Returns the number of timeseries schemas being used.
+    async fn get_schema_count(client: &Client) -> usize {
+        client
+            .execute_with_body(
+                "SELECT * FROM oximeter.timeseries_schema FORMAT JSONEachRow;",
+            )
+            .await
+            .expect("Failed to SELECT from database")
+            .lines()
+            .count()
+    }
+
+    #[tokio::test]
+    async fn test_database_version_update_idempotent() {
+        let log = slog::Logger::root(slog::Discard, o!());
```

Comment:
I've been trying to move all the tests over to the test logging infrastructure from `omicron_test_utils`. The original tests predated all that stuff, I think, or I didn't use it for some reason.

Would you mind using the `test_setup_log` function a la L1312?

Reply:
Sounds good, done for this and for several other callers in this file.